lib/services/ios-debugger-port-service.ts

```diff
@@ -58,13 +58,26 @@ export class IOSDebuggerPortService implements IIOSDebuggerPortService {
 
 	@cache()
 	private attachToDebuggerPortFoundEventCore(): void {
-		this.$iOSLogParserService.on(DEBUGGER_PORT_FOUND_EVENT_NAME, (data: IIOSDebuggerPortData) => {
-			this.$logger.trace(DEBUGGER_PORT_FOUND_EVENT_NAME, data);
-			this.setData(data, { port: data.port });
-			this.clearTimeout(data);
+		this.$logParserService.addParseRule({
+			regex: IOSDebuggerPortService.DEBUG_PORT_LOG_REGEX,
+			handler: this.handlePortFound.bind(this),
+			name: "debugPort",
+			platform: this.$devicePlatformsConstants.iOS.toLowerCase()
 		});
 	}
 
+	private handlePortFound(matches: RegExpMatchArray, deviceId: string): void {
+		const data = {
+			port: parseInt(matches[1]),
+			appId: matches[2],
+			deviceId,
+			projectDir: ""
```

Comment:
Why this is empty? Not sure if this will be ok for sidekick. Maybe we need to use
```
projectDir: this.$projectDataService.getProjectData().projectDir
```

or
```
projectDir: this.$projectData.projectDir
```

Reply:
This project data might not be valid in Sidekick. Moreover until now this property was totally missing. Maybe it is better to set it to null/undefined.